pavelib/utils/test/suites/nose_suite.py

```diff
@@ -21,6 +21,15 @@ def __init__(self, *args, **kwargs):
         self.fail_fast = kwargs.get('fail_fast', False)
         self.run_under_coverage = kwargs.get('with_coverage', True)
         self.report_dir = Env.REPORT_DIR / self.root
+
+        # If set, put reports for run in "unique" directories.
+        # The main purpose of this is to ensure that the reports can be 'slurped'
+        # in the main jenkins flow job without overwriting the reports from other
+        # build steps. For local development/testing, this shouldn't be needed.
+        if os.environ.get("SHARD", None):
```

Comment:
will this return False if the env var is not set? (Put differently, if we don't have the env var set, will this result in some directories being "shard_None"?)